components/try.tsx

```diff
@@ -13,28 +16,63 @@ export default function TryPear() {
         </div>
 
         {/* Content area */}
-        <div className="relative pb-10 pt-32 md:pb-16 md:pt-40">
+        <div className="relative pb-2 pt-32 md:pb-4 md:pt-40">
           {/* Section header */}
           <div className="mx-auto max-w-3xl pb-12 text-center md:pb-16">
-            <h1 className="h1 mb-4" data-aos="fade-up">
-              Try PearAI For Free
+            <h1 className="mb-4 text-4xl font-bold" data-aos="fade-up">
```

Comment:
```suggestion
            <h1 className="mb-4 text-4xl font-semibold" data-aos="fade-up">
```
change the thickness of the header font to match figma and hero